main.go

```diff
@@ -40,7 +40,7 @@ func main() {
 		glog.Exitf("cloudprober could not be initialised from config: grpc_port: %d, err:%v", *rpcPort, err)
 	}
 
-	ctx, hermes.CancelCloudprober = context.WithCancel(context.Background())
+	ctx := context.Background()
```

Comment:
Nit: It's fine use context.Background() as is. No need to declare a variable.

Reply:
Great, I'll make this change.